userguide-book.ditamap

```diff
@@ -55,7 +55,9 @@
       <topicref keyref="web-based-resources" print="yes"/>
       <topicref keyref="books"/>
     </appendix>
-    <appendix keyref="release-notes"/>
+    <appendix keyref="release-notes">
+      <ditavalref href="resources/pdf.ditaval"/>
+    </appendix>
```

Comment:
This branch filtering approach would only apply to the _Release Notes_.

While that's fine now, there may be cases in the future where we'd like to filter other topics by `deliveryTarget` to exclude things from PDF output, so I'll move the filter reference to the PDF build target.